src/store.ts

```diff
@@ -45,13 +45,24 @@ export class Store<T> {
 
   private dispatchQueue: DispatchQueueItem<T>[] = [];
 
+  private enableDevTools: boolean = true;
+
   constructor(private initialState: T, options?: Partial<StoreOptions>) {
     this.options = options || {};
     const isUndoable = this.options.history && this.options.history.undoable === true;
     this._state = new BehaviorSubject<T>(initialState);
     this.state = this._state.asObservable();
+    
+    // Do we have options, as well as devToolsOptions and the disable option on the options object?
+    if (typeof this.options.devToolsOptions !== "undefined" && typeof this.options.devToolsOptions.disable !== "undefined") {

```

Comment:
might switch to
```
if (this.options.devToolsOptions && this.options.devToolsOptions.disable === false)
```

to capture null, empty space and all sorts of other weird situations.

Reply:
Good idea. Changed.